.github/workflows/pr-checks.yml

```diff
@@ -13,7 +13,7 @@ jobs:
         # For reproducibility, we define ubuntu-20.04 rather than ubuntu-latest. PR-CHECKS should only test default.
         os: [ ubuntu-20.04 ]
         # PR-CHECKS should only test default python distribution.
-        python-version: [ '3.11' ]
+        python-version: [ '3.11', '3.12' ]
```

Comment:
Will remove this change once checks pass. Just want to force 3.12 right now.

Reply:
It passed: https://github.com/causalens/cai-causal-graph/actions/runs/6590490817